### PR TITLE
Update the mono-repo to one package

### PR DESCRIPTION
**Pull request by abdonrd**
We want to try to publish the components as a single package in npm.

### CODE REVIEW

**Comment by abdonrd** [comment]
Right now the Storybook is not working:

Screenshot 2021-11-08 at 18 44 31

**Comment by abdonrd** [comment]
Seems that the problem is the resolution of the dependencies. We need to wait to the next `carbon-web-components` release, which stops having dependencies as `peerDependencies`.

File: package.json
Comment (abdonrd) [inline package.json:37]: Also remove `carbon-components` when update `carbon-web-components`.

File: package.json
Comment (abdonrd) [inline package.json:37]: Wait for the release before merging the PR.

File: package.json
Comment (abdonrd) [reply package.json]: > Right now the Storybook has stopped working:
> 
> Screenshot 2021-11-08 at 18 44 31

It's working now! 🎉

**Comment by abdonrd** [comment]
> Right now the Storybook has stopped working:
> 
> Screenshot 2021-11-08 at 18 44 31

It's working now! 🎉

**Review by Tansito — APPROVED** [review]

**Pull request merged**
